src/components/input-field/input-field.tsx

```diff
@@ -300,6 +328,35 @@ export class InputField {
         );
     }
 
+    private renderTrailingErrorIcon() {
+        if (
+            !this.trailingErrorIcon ||
+            (this.value && this.value.length >= this.minlength) ||
+            !this.value
+        ) {
+            return;
+        }
+
+        return (
+            <i
+                onKeyPress={this.handleIconKeyPress}
+                onClick={this.handleIconClick}
+                class={`
+                    ${
+                        this.isFocused
+                            ? 'mdc-text-field-icon-color'
+                            : 'mdc-text-field__icon'
+                    }
+                
+                `}
```

Comment:
It's possible to write this in a nicer way using objects instead of string literals :slightly_smiling_face: 
```jsx
<i class={{
    'this-class-is-applied': true,
    'this-class-is-not-applied': false
}} />
```